skbio/tree/_tree.py

```diff
@@ -2717,6 +2717,52 @@ def compare_tip_distances(self, other, sample=None, dist_f=distance_from_r,
         return dist_f(self_matrix, other_matrix)
 
     @experimental(as_of="0.4.0")
+    def bifurcate(self, insert_length=None, remove_singles=True):
+        r"""Reorders the tree into a bifurcating tree, so that
+        every internal node will have at most two children.
+
+        All nodes that have more than 2 children will
+        have additional intermediate nodes inserted to ensure that
+        every node has only 2 children.
+
+        Parameters
+        ----------
+        insert_length : int, optional
+            The branch length assigned to all inserted nodes.
+
+        remove_singles : bool, optional
```

Comment:
All this parameter does is call `prune()`. Would users expect that singles are removed by default? If not, I think we can remove this parameter and add a mention in the Notes section about calling `prune()` if the user wants to remove singles.